wGetStarted/MacroSecurity.md

```diff
@@ -0,0 +1,52 @@
+### TEST 16: Drill To An Altered Macro File
+
+This drill is the same as the previous two tests. However, you will setup the TargetDrill file as a link/network file and then alter it before drilling to it.
+
+**Actions:**
+
+*  Login to Interject
+*  Upload and setup drill for report
+*  Alter the TargetDrill file
+*  Open "ReportDrill.xlsx" file
+*  Run drill
+
+**Step 1:** Login to Interject with the following credentials:
+
+![](/images/Testing/LoginAdminS.png)
+<br>
+
+**Step 2:** Open the Report Library and create a new link:
+
+![](/images/Testing/UploadTargetDrillLink.png)
+<br>
+
+**Step 3:** Click &lt;Browse&gt; and navigate to "STargetDrillLink.xlsm" and click &lt;Open&gt; and then &lt;Save&gt; the link:
```

Comment:
using angle brackets on `<Open>` and `<Save>` refering to them as buttons is different from some other pages that just use bold. might want to make these or those consistent.